configs/components/ruby.rb

```diff
@@ -4,9 +4,9 @@
   pkg.url "https://cache.ruby-lang.org/pub/ruby/2.3/ruby-#{pkg.get_version}.tar.gz"
 
   if platform.is_windows?
-    pkg.add_source "http://buildsources.delivery.puppetlabs.net/windows/elevate/elevate.exe", sum: "bd81807a5c13da32dd2a7157f66fa55d"
+    pkg.add_source "file://resources/files/windows/elevate.exe", sum: "bd81807a5c13da32dd2a7157f66fa55d"
```

Comment:
I don't remember if we looked at licensing for this. Can we redistribute it? At this point, it's part of vanagon, which feels weird to me.